runtime-config-linux.md

```diff
@@ -34,32 +47,27 @@ Also, when a path is specified, a runtime MUST assume that the setup for that pa
     ]
 ```
 
-#### Namespace types
+## Devices
 
-* **pid** processes inside the container will only be able to see other processes inside the same container.
-* **network** the container will have its own network stack.
-* **mount** the container will have an isolated mount table.
-* **ipc** processes inside the container will only be able to communicate to other processes inside the same
-container via system level IPC.
-* **uts** the container will be able to have its own hostname and domain name.
-* **user** the container will be able to remap user and group IDs from the host to local users and groups
-within the container.
+`devices` is an array specifying the list of devices to be created in the container.
 
-## Devices
+The following parameters can be specified:
+
+* **`path`** *(string, required)* - full path to device inside container
+
+* **`type`** *(char, required)* - type of device: `c`, `b`, `u` or `p`. More info in `man mknod`.
 
-Devices is an array specifying the list of devices to be created in the container.
-Next parameters can be specified:
+* **`major, minor`** *(int64, optional)* - major, minor numbers for device. More info in `man mknod`. There is a special value: `-1`, which means `*` for `device` cgroup setup.
 
-* **type** - type of device: `c`, `b`, `u` or `p`. More info in `man mknod`
-* **path** - full path to device inside container
-* **major, minor** - major, minor numbers for device. More info in `man mknod`.
-                 There is special value: `-1`, which means `*` for `device`
-                 cgroup setup.
-* **permissions** - cgroup permissions for device. A composition of `r`
-                (read), `w` (write), and `m` (mknod).
-* **fileMode** - file mode for device file
-* **uid** - uid of device owner
-* **gid** - gid of device owner
+* **`permissions`** *(string, optional)* - cgroup permissions for device. A composition of `r` (*read*), `w` (*write*), and `m` (*mknod*).
+
+* **`fileMode`** *(uint32, optional)* - file mode for device file
+
+* **`uid`** *(uint32, optional)* - uid of device owner
+
+* **`gid`** *(uint32, optional)* - gid of device owner
```

Comment:
On Wed, Sep 23, 2015 at 09:57:22AM -0700, Antonio Murdaca wrote:
> I'll fix these inconsistencies by documenting them as per #101
> rather than not adding required|optional, sounds good?

It's probably best to leave the #101 docs in that PR to avoid stalling
this one for the same reasons that have stalled #101 (I'm not sure
what those reasons are).  So I'd recommend making as few changes to
the existing text in this section as possible to avoid controversy.